### ISSUE TITLE
Improve UI for 'Welcome' screen

### ISSUE THREAD

**Issue by saramakishti** [amosproj/amos2024ws04-personalized-play#53]
### Purpose
We need to align the design specifications on all app screens.  

### User Story
As a user, I want the UI design to be consistent so that I find the app engaging.

### Acceptance criteria

- [x] All font is 'Inter'
- [x] All buttons have rounded corners 12px (all corners)
- [x] The image on center of the screen will be replaced with an image that will be provided by the POs after consultation with IP

There is a big heading in the center of the screen:
- [x] Text: 'Welcome to MUMBI!'
- [x] Font weight is: bold
- [x] Text color is: #620674

There is a subheading that has:
- [x] Text: 'We are going to ask a few questions before we can suggest a personalized play'

There is a primary button under the subheading:
- [x] Text: 'Get started'
- [x] Background color is: #620674
- [x] Text color is: #ffffff

There is a secondary button under the primary button:
- [x] Text: 'Skip for now'
- [x] Background color is: #B183BA
- [x] Text color is: #ffffff
- [x] Clicking on the primary button will start the onboarding flow
- [x] Clicking on the secondary button will take the user to the app's home screen (onboarding will be skipped)

### Design notes

* Wireframes are [here](https://app.visily.ai/projects/c005caa3-974a-4e5b-be6c-2b00e5b6645b/boards/1382078)

* Design specifics are [here](https://app.visily.ai/projects/c005caa3-974a-4e5b-be6c-2b00e5b6645b/boards/1436412)

---

### Definition of Done (DoD)

- [ ] The feature has been fully implemented, thoroughly tested, and integrated into the app with no critical bugs or performance issues.
- [ ] Manual testing is complete, verifying that the feature meets all acceptance criteria.
- [ ] Big features are documented inside the code, with clear explanations of functionality, usage, and any relevant edge cases.
- [ ] The feature has undergone peer review, with the code approved by at least one other team member to ensure quality and maintainability.
- [ ] The feature branch has been successfully merged into the main branch, and the pull request is closed and the branch is deleted.
- [ ] The feature’s UI/UX has been reviewed to ensure consistency and alignment with design guidelines.

**Comment by CaioSilvaUT** [comment]
Hello, I'm a student from UTFPR in Brazil, and I'm taking a class where I need to contribute to open source issues. I would like to know if I can take on this issue to work on.

**Comment by saramakishti** [comment]
@CaioSilvaUT Hello!
 
This project (and issue) is part of a university course known as the 'AMOS Project' held with TU Berlin and FAU universities in Germany. Although it is open source, we are currently working on this as a team and will be graded for the completion of these issues, therefore I don't think you can work on this without being enrolled in the course. 

However, I highly suggest you read more about the [AMOS Project here](https://oss.cs.fau.de/2024/10/17/the-amos-project-winter-2024-25/) and directly contact the organizer of this course: dirk.riehle@fau.de as he might guide you in what you are looking for.

Wish you all the best :)

**Comment by yashbhesaniya** [comment]
![Image](https://github.com/user-attachments/assets/06b61802-0df8-44e8-9c50-1677070d53c7)
